Maestro/subscriptions.json

```diff
@@ -6,6 +6,11 @@
       "vsoProject": "DevDiv",
       "buildDefinitionId": 8550
     },
+    "aspnet-dependency-update": {
```

Comment:
Could you add a quick comment, like `// A build definition capable of running any .ps1 script in the aspnet/universe repo`?